content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -662,53 +662,99 @@ updates:
+            <td rowspan="3" align="center"><code>2.0.0</code></td>
+            <td align="center"><code>'widen'</code></td>
+            <td><code>&gt;=1.0.0 &lt;3.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase-if-necessary'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+    </tbody>
+</table>
 
 Use the `versioning-strategy` option to change this behavior for supported package managers.
 
 {% data reusables.dependabot.option-affects-security-updates %}
 
 Available update strategies
 
-| Option | Supported by | Action |
-|--------|--------------|--------|
-| `lockfile-only` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Only create pull requests to update lockfiles. Ignore any new versions that would require package manifest changes. |
-| `auto` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Follow the default strategy described above.|
-| `widen`| `composer`, `npm` | Relax the version requirement to include both the new and old version, when possible. |
-| `increase`| `bundler`, `composer`, `npm` | Always increase the version requirement to match the new version. |
-| `increase-if-necessary` | `bundler`, `composer`, `npm` | Increase the version requirement only when required by the new version. |
+| Ecosystem | Supported _versioning-strategies_ | Default strategy | Additional notes |
+|-----------|-----------------------------------|------------------|------------------|
+| `bundler` | `auto`, `increase`, `increase-if-necessary`, `lockfile-only` | `auto` |
+| `cargo` | `auto`, `lockfile-only` | `auto` |
+| `composer` | `auto`, `lockfile-only`, `widen`, | `auto` |
+| `docker` | TODO | TODO |
+| `github-actions` | TODO | TODO |
+| `gitsubmodule` | TODO | TODO |
+| `go` | TODO | TODO |
+| `gradle` | TODO | TODO |
+| `maven` | TODO | TODO |
+| `mix` | `auto`, `lockfile-only` | `auto` |
+| `npm` | `auto`, `lockfile-only`, `widen` | `auto` |
+| `nuget` | TODO | TODO |
+| `pip` | `auto`, `lockfile-only` | `auto` |
+| `pub` | TODO | TODO |
+| `terraform` | TODO | TODO |
+
+**Note:** The strategy code is open source, so if you'd like a particular ecosystem to support a new strategy, you are always welcome to submit a PR to https://github.com/dependabot/dependabot-core/.
 
 ```yaml
-# Customize the manifest version strategy
+# Example config for customizing the manifest version strategy
 
 version: 2
 updates:
-  - package-ecosystem: "npm"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    # Update the npm manifest file to relax
-    # the version requirements
-    versioning-strategy: widen
-
   - package-ecosystem: "composer"
     directory: "/"
     schedule:
-      interval: "daily"
-    # Increase the version requirements for Composer
-    # only when required
+      interval: "weekly"
```

Comment:
Changing the interval was intentional... see https://github.com/github/docs/pull/20734/